apps/web-giddh/src/app/settings/permissions/form/form.component.html

```diff
@@ -1,143 +1,219 @@
 <form name="permissionForm" [formGroup]="permissionForm" [ngClass]="{'permissionForm': isHorizntl}"
     (ngSubmit)="submitPermissionForm()">
 
-    <div class="form-group" style="margin-left:0;">
-        <label *ngIf="isLblShow">Unique Name/Email ID</label>
-        <input type="text" name="emailId" formControlName="emailId" placeholder="Unique Name/Email ID"
-            class="form-control" [attr.readonly]="userdata?.uniqueName" />
+    <div class="row mb-15">
+        <div class="col-sm-6">
+           <div class="form-group">
+                <label *ngIf="isLblShow">Unique Name/Email ID</label>
+                <input type="text" name="emailId" formControlName="emailId" placeholder="Unique Name/Email ID"
+                    class="form-control" [attr.readonly]="userdata?.uniqueName" />
+            </div>
+        </div>
+        <div class="col-sm-6">
+            <div class="form-group">
+                <label *ngIf="isLblShow">Role Name</label>
+                <!-- <select name="roleUniqueName" formControlName="roleUniqueName" class="form-control ">
+                <option [value]="role.uniqueName" *ngFor="let role of allRoles">{{role.name}}</option>
+            </select> -->
+                <sh-select placeholder="Select role" formControlName="roleUniqueName" [options]="allRoles"></sh-select>
+            </div>
+        </div>
     </div>
 
-    <div class="form-group">
-        <label *ngIf="isLblShow">Role Name</label>
-        <!-- <select name="roleUniqueName" formControlName="roleUniqueName" class="form-control ">
-        <option [value]="role.uniqueName" *ngFor="let role of allRoles">{{role.name}}</option>
-    </select> -->
-        <sh-select placeholder="Select role" formControlName="roleUniqueName" [options]="allRoles"></sh-select>
+    <div class="clearfix">
+        <p class="bold font-14 mb-1 d-inline-block middle mr-1">Specific Permission</p>
```

Comment:
Add to JSON

Reply:
Already add JSON for **Specific Permission** content